lookout/style/format/features.py

```diff
@@ -30,14 +30,15 @@ class FeatureType(enum.Enum):
 
 class VirtualNode:
     def __init__(self,  value: str, start: Position, end: Position,
-                 *, node: bblfsh.Node = None, y: int = None):
+                 *, node: bblfsh.Node = None, y: int = None, path: str = None):
```

Comment:
Should be compulsory argument if we set it everywhere and rely on it later on.

Reply:
TBH - it used to work before without it. I'm using it for analysis - I'm not sure if it should be mandatory

Reply:
@EgorBu Do you mean we do not access this attribute anywhere in the code?

Reply:
Done.